ynr/apps/moderation_queue/templates/moderation_queue/photo-review-list.html

```diff
@@ -70,6 +71,7 @@ <h1>Uploaded photos for review</h1>
           <td>{{ queued_image.election_date }}</td>
           <td>{% if queued_image.user %}{{ queued_image.user.username }}{% else %}a robot 🤖{% endif %}</td>
           <td><a href="{% url 'person-view' person_id=queued_image.person.id %}">{{ queued_image.person.id }}</a></td>
+          <td>{% if queued_image.detection_metadata %}Photo Angle: {{ queued_image.photo_angle }}{% else %}Image analysis is not available for this photo.{% endif %}</td>
```

Comment:
I've added this as a placeholder, but what would be useful to display here? Do we want to display the entire (collapsible)`dict` or ?

Reply:
Resolved in https://github.com/DemocracyClub/yournextrepresentative/pull/2015/commits/8d93e511cd21c0057a08b417fdd083447a2733dd